infra/cifuzz/fuzz_target.py

```diff
@@ -121,6 +124,34 @@ def is_reproducible(self, test_case):
         return True
     return False
 
+  def is_crash_valid(self, test_case):
+    """Checks if a crash is reproducible, and if it is, whether it's a new
+    regression that cannot be reproduced with the latest OSS-Fuzz build.
+
+    NOTE: If no old_build_target is specified the crash is assumed introduced
+    by the pull request if it is reproducible.
+
+    Args:
+      test_case: The path to the test_case that triggered the crash.
+
+    Returns:
+      True if the crash was introduced by the current pull request.
+    """
+    exists_in_pr = self.is_reproducible(test_case,
```

Comment:
Let's use the term "reproducible" instead of "exists" if it makes sense to talk this way instead of "novel"